include/openssl/evp.h

```diff
@@ -804,6 +804,10 @@ __owur int EVP_CipherInit_ex(EVP_CIPHER_CTX *ctx,
                                  const EVP_CIPHER *cipher, ENGINE *impl,
                                  const unsigned char *key,
                                  const unsigned char *iv, int enc);
+__owur int EVP_CipherInit_skey(EVP_CIPHER_CTX *ctx, const EVP_CIPHER *cipher,
```

Comment:
EVP_CipherInit_SKEY to make it similar to the other inits with SKEY. Or change the others to lowercase skey. In do not have a strong preference for either.

Reply:
Done, thanks